test/input_cases/battery_common_data.h

```diff
@@ -90,11 +90,11 @@ namespace {
 		ssc_number_t p_replacement_cost[1] = { 68 };
 		ssc_data_set_array(data, "om_replacement_cost1", p_replacement_cost, 1);
 		ssc_data_set_number(data, "batt_mass", 3102.717041015625);
-		ssc_data_set_number(data, "batt_length", 1.0689570903778076);
-		ssc_data_set_number(data, "batt_width", 1.0689570903778076);
-		ssc_data_set_number(data, "batt_height", 1.0689570903778076);
-		ssc_data_set_number(data, "batt_Cp", 1004);
-		ssc_data_set_number(data, "batt_h_to_ambient", 5);
+		ssc_data_set_number(data, "batt_length", 7);
```

Comment:
why did these inputs change?

Reply:
Using the new thermal model, when the room temperature doesn't change, the battery temperature becomes primarily a function of the source term, I^2*R / surf_area / h. 

With the inputs provided here, the battery dimensions are too small for its power. It's current can go up to 3037 A for instance (batt_current_charge_max) and so it was overheating the battery way too much.

Reply:
So I imagine it was causing the test to fail, but only because it was a highly non-physical test to begin with?

Reply:
I think so